Modules/Loadable/Markups/MRML/vtkMRMLMarkupsPlaneNode.cxx

```diff
@@ -789,26 +789,26 @@ vtkMatrix4x4* vtkMRMLMarkupsPlaneNode::GetPlaneToPlaneOffsetMatrix()
 //---------------------------------------------------------------------------
 void vtkMRMLMarkupsPlaneNode::UpdateInteractionHandleToWorldMatrix()
 {
-  double handleX_World[3] = { 0.0 };
-  double handleY_World[3] = { 0.0 };
-  double handleZ_World[3] = { 0.0 };
+  double handleX_World[3] = { 0.0, 0.0, 0.0 };
```

Comment:
Does the change make any functional different or is it only for making it clear what the code does exactly? (I thought that `={0.0}` initializes each element to 0.0)

Reply:
It's just for clarity. I seem to recall from a previous PR that this was the preferred style for array initialization.

Reply:
OK, I just wanted to make sure that `={0}` was not wrong.